dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -100,6 +103,10 @@ def unauthorized_status
     render(status: :unauthorized, json: {error: 'Unauthorized'})
   end
 
+  def error_status
```

Comment:
Tiny nit: should this method name be more specific to the kind of error? I'm also fine leaving it like this for now, and changing it later if we have another kind of error we need to handle.